lib/fetch/sanitize.js

```diff
@@ -155,6 +155,7 @@ export function sanitizeBadger(json) {
     linkedin: pathOr(null, ['url'], get('badger.linkedin')),
     github: pathOr(null, ['url'], get('badger.github')),
     twitter: pathOr(null, ['url'], get('badger.twitter')),
+    squarespaceId: get('badger.squarespace-author-id'),
```

Comment:
Totally not a big deal but as a nicety we could keep `get`s with `get`s and `pathOr`s with `pathOr`s 🤠